### DISCUSSION TITLE
Alternative Mobile menu

### DISCUSSION THREAD

**Discussion by finalwebsites** [bootscore/bootscore#833] [Ideas]
Hi,

just for simple menu's it would be great to have an alternative menu for mobile phones.
Themes like OceanWP have this kind of features (I didn't checked is this comes together with a walker menu of that it's just a second menu slot)

**Comment by crftwrk** [comment]
What do you mean exactly with an alternate menu? Have a link to OceanWP‘s menu demo?

**Reply by finalwebsites** [reply]
Check this site https://www.handiggoed.nl/

**Reply by crftwrk** [reply]
Do you mean the collapse menu instead the offcanvas?

**Reply by finalwebsites** [reply]
> Do you mean the collapse menu instead the offcanvas?
You're right, you solved this using the offcanvas menu. This way you can show any menu using a widget.
I think I should go for it this way, thanks.
